docs/source/search/granules.ipynb

```diff
@@ -0,0 +1,404 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Searching by Additional Attributes\n",
+    "\n",
+    "The MAAP platform has built upon the base CMR database and provided additional attributes to the granule metadata so that users can search for specific information. There are many additional attributes available. To get started users can search by the following keywords:\n",
+    "\n",
+    "* `site_name`\n",
+    "* `data_format`\n",
+    "* `track_number`\n",
+    "* `polarization`\n",
+    "\n",
+    "For example, if a user is only interested in using data from the _Mondah Forest Gabon_ site with a data format of _GeoTIFF_ we can use the following query:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'Got 18 results'\n"
+     ]
+    }
+   ],
+   "source": [
+    "results = maap.searchGranule(site_name=\"Mondah Forest Gabon\",data_format='GeoTIFF')\n",
+    "pprint(f'Got {len(results)} results')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'Granule': {'AdditionalAttributes': {...},\n",
+      "             'Collection': {...},\n",
+      "             'DataFormat': 'GeoTIFF',\n",
+      "             'DataGranule': {...},\n",
+      "             'GranuleUR': 'AfriSAR_AGB_Maps.Mondah_AGB_50m.tif',\n",
+      "             'InsertTime': '2019-05-10T00:00:00Z',\n",
+      "             'LastUpdate': '2019-05-13T12:45:07Z',\n",
+      "             'MeasuredParameters': {...},\n",
+      "             'OnlineAccessURLs': {...},\n",
+      "             'OnlineResources': {...},\n",
+      "             'Orderable': 'false',\n",
+      "             'Spatial': {...},\n",
+      "             'Temporal': {...},\n",
+      "             'Visible': 'true'},\n",
+      " 'collection-concept-id': 'C1200115768-NASA_MAAP',\n",
+      " 'concept-id': 'G1200115791-NASA_MAAP',\n",
+      " 'format': 'application/echo10+xml',\n",
+      " 'revision-id': '2'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "pprint(results[0],depth=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The returned results will give you only GeoTIFF datasets that have been tagged as part of the Mondah Forest Gabon research site.\n",
+    "\n",
+    "**DISCLAIMER:** The MAAP data team is working to update the addtional attributes within the MAAP platform so these are subject to change. Furthermore, the accepted parameters for the addtional attributes are changing and further documentation will be provided as updates come."
```

Comment:
"Additional" is misspelled twice in this paragraph.

Reply:
Just pushed updated. Please verify and resolve